doc/docker.md

```diff
@@ -0,0 +1,189 @@
+# Docker info
+
+The main readme contains the basic steps to start with Docker:
+
+    git clone git@github.com:concord-consortium/rigse.git portal
+    cd portal
+    docker-compose up # this will take 15 minutes to download gems
+
+After this you can browse to [http://0.0.0.0:3000](http://0.0.0.0:3000). On OS X this
+might take more than 5 minutes to load the first page. Look in the terminal where you ran
+`docker-compose up` to monitor progress.
+
+## Notes on basic steps
+
+If you have a clean checkout of the portal, `docker-compose up` should just work. If you
+have another service running on port 3000 you might get a port conflict.
+
+By default your local portal folder will be mounted inside of the docker app service.
+Any changes you make to your local folder will show up immediately in the docker app
+service. The files `config/database.yml`, `config/settings.yml` and
+`config/app_environment_variables.yml` are automatically copied from their `.sample.yml`
+counterparts when you run `docker-compose up` if they do not exist. If they already
+exist, they will not be updated. If they already exist then there is a good chance
+they will not be configured correctly for Docker. You can delete these files and run
+`docker-compose restart` which will fix things.  
+
+Also, when `config/database.yml` is not present yet, `docker-compose up` will copy it
+and run `rake db:setup`. `rake db:setup` erases any data in your database, so be careful.
```

Comment:
It will be DB created by docker-compose, not the local one, so maybe we could mention that.

Also, all these things with files copying are confusing to me in the long term (even though it was me who I implemented that AFAIR). I never know what is done in my local filesystem and what is done in the container, and I had some issues with that. I guess that if you start from clean repo, it'll work as expected - the config files will be copied. But I'm not sure that if you remove those config files locally and run docker-compose again, it'll work as expected. I think the files will be there already.